pkgs/by-name/te/tekens-streamlabels/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "tekens-streamlabels";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "teken";
+    repo = "streamlabels";
+    rev = "v${version}";
+    hash = "sha256-FAu00SucWHMQbhp9mJYIdPWtbiny09LABYkeZdV10GQ=";
+  };
+
+  vendorHash = "sha256-eZQe8gstzj9BxjuZiXV1UFnuzjICJBFjiTpJNGFiyFI=";
+
+  meta = with lib; {
+    description = "A small go app to get basic stream labels for twitch";
+    longDescription = ''
+      A simple Go program to generate text files containing Twitch stream information like newest followers, subscribers, and bits leaderboard. 
+      These files can then be used as sources in OBS Studio or other streaming software.
+    '';
+    homepage = "https://github.com/teken/streamlabels";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ "teken" ];
+  };
```

```suggestion
  meta = {
    description = "A small go app to get basic stream labels for twitch";
    longDescription = ''
      A simple Go program to generate text files containing Twitch stream information like newest followers, subscribers, and bits leaderboard. 
      These files can then be used as sources in OBS Studio or other streaming software.
    '';
    homepage = "https://github.com/teken/streamlabels";
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [
      teken
    ];
  };
```
1. `lib` is a large scope, and `with`-ing it in large quantities slows down eval.
2. Maintainers are variables in `/maintainers/maintainer-list.nix`, add yourself there.